ui/apps/platform/src/Containers/Clusters/ClusterEditForm.tsx

```diff
@@ -50,6 +51,19 @@ function ClusterEditForm({
                     clusterRetentionInfo={clusterRetentionInfo}
                     isManagerTypeNonConfigurable={isManagerTypeNonConfigurable}
                 />
+            ) : (
+                <Alert variant="warning" isInline title="Legacy installation method" component="p">
+                    <p>
+                        Legacy installation is not recommended. It causes extra operational
```

Comment:
If we wanted to rewrite to comply better with style guidelines, we could say something like:

To avoid extra operational complexity, use a cluster init bundle with either of the following installation methods:
Operator for Red Hat OpenShift
Helm chart for other platforms

Only use the legacy installation method if you have a specific installation need that requires using this method.


(I lifted the language in the last sentence from the docs: https://docs.openshift.com/acs/4.3/installing/installing_ocp/install-central-ocp.html#install-using-roxctl-ocp)

Reply:
Thank you very much. Here is the updated picture:
![Rewrite_with_fix](https://github.com/stackrox/stackrox/assets/11862657/75db4b3e-2886-44f9-a16f-ca42c3852d81)